linux_os/guide/system/accounts/accounts-restrictions/password_storage/no_empty_passwords/bash/shared.sh

```diff
@@ -11,8 +11,9 @@ for FILE in ${NULLOK_FILES}; do
    sed --follow-symlinks -i 's/\<nullok\>//g' ${FILE}
 done
 {{% elif 'ubuntu' in product %}}
-FILE="/etc/pam.d/common-password"
-sed -i 's/\(.*pam_unix\.so.*\)\s\<nullok\>\(.*\)/\1\2/g' ${FILE}
+{{{ bash_pam_unix_enable() }}}
+sed --follow-symlinks -i 's/\<nullok\>//g' /usr/share/pam-configs/cac_unix
```

Comment:
no_empty_passwords rule is also used in Ubuntu STIGs, which state to remove `nullok` only from `/etc/pam.d/common-password`. https://www.stigviewer.com/stig/canonical_ubuntu_22.04_lts/2024-05-30/finding/V-260570

I suggest that for backwards compatibility, we only remove nullok from the `Passwords` section in `cac_unix`.

Reply:
Resolved by https://github.com/ComplianceAsCode/content/blob/aa92d0eb8330d90c8ff3e4ac38d0c3cbfaf03fd5/shared/macros/10-bash.jinja#L864-L866